### ISSUE TITLE
[gatsby-source-wordpress] Issues with images and sharp errors

### ISSUE THREAD

**Issue by AnalogMemory** [gatsbyjs/gatsby#4846]
Past week I've noticed some odd errors with the sharp plugin
While it's generating thumbnails I'll get 2-3 errors like this

```
Generating image thumbnails [=========] 93/97 12.9 secs 96%
(sharp:11982): GLib-GObject-WARNING **: 
invalid uninstantiatable type '(null)' in cast to 'GObject'
(sharp:11982): GLib-GObject-CRITICAL **: 
g_object_set_qdata: assertion 'G_IS_OBJECT (object)' failed
(sharp:11982): GLib-GObject-WARNING **: 
invalid uninstantiatable type '(null)' in cast to 'GObject'
(sharp:11982): GLib-GObject-CRITICAL **: 
g_object_set_qdata: assertion 'G_IS_OBJECT (object)' failed
Generating image thumbnails [========] 97/97 16.5 secs 100%
```

After this one or more of the pages will have missing images.

It feels like it's an issue with the image/jpg but I can't figure out what the exact problem is. If i swap out the problem image with a image known to work the problem goes away. 

### Updated 
So it seems images uploaded in the past week are the issue but I can't figure out why?
The data for the images are being pulled in but then sharp isn't processing them. Localfile returns null

```
{
  "data": {
    "wordpressPage": {
      "title": "Testing",
      "id": "f85fba23-9a7b-5eee-a77b-b937e1b267c6",
      "acf": {
        "sections_page": [
          {
            "__typename": "WordPressAcf_hero",
            "hero_background_image": [
              {
                "id": "784ee461-cec7-515c-acc2-e85ec4fd7efa",
                "media_type": "image",
                "media_details": {
                  "width": 3000,
                  "height": 1916,
                  "file": "test.jpg"
                },
                "mime_type": "image/jpeg",
                "localFile": null
              }
            ],
...
```

Data is coming in from a self-hosted WordPress build with ACF fields

Gatsby related plugins
```
gatsby: ^1.9.246
gatsby-image: ^1.0.43
gatsby-link: ^1.6.40
gatsby-paginate: ^1.0.13
gatsby-plugin-react-helmet: ^2.0.10
gatsby-plugin-react-next: ^1.0.11
gatsby-plugin-sass: ^1.0.25
gatsby-plugin-sharp: ^1.6.41
gatsby-plugin-styled-components: ^2.0.11
gatsby-source-filesystem: ^1.5.28
gatsby-source-wordpress: ^2.0.73
gatsby-transformer-sharp: ^1.6.22
```

**Comment by sebastienfi** [comment]
May it be a corrupted file ? I would open the image with an ASCII editor and check that the first bytes of data coincide with the file extension.

Are you on Unix ?

**Comment by AnalogMemory** [comment]
That's what I thought at first and tried to narrow it down to where it works and then doesn't. 

Any new images added and approx the last 15 images uploaded refuse to get processed it seems. I've  added the same "problem images" to a similar build and they all works just fine. Which makes me think it's some sort of config issue.

**Comment by eladams** [comment]
I'm experiencing the same issue, any solution?

**Comment by AnalogMemory** [comment]
@eladams my theory is that it was some sort of naming collision? I did two things to that seemed to help. 

1: I included the id for each object in the ACF image object.
2: I gave each image object a unique name (e.g `team_photo`). All my image ACF fields were named photo, so doing this made it a bit more clearer

Not sure if any of these will help you, but it seemed to fix it for me after I updated all the GraphQL queries to something like below.

```
node {
  id
  title
  acf {
    team_photo: photo {
      id
      localFile {
        id
        childImageSharp {
          resize(width: 800, height: 800) {
            src
          }
        }
      }
    }
  }
}
```
